src/hooks/useRemainingRelays.ts

```diff
@@ -4,7 +4,7 @@ import useSafeAddress from '@/hooks/useSafeAddress'
 import { Errors, logError } from '@/services/exceptions'
 import { SAFE_GELATO_RELAY_SERVICE_URL } from '@/config/constants'
 
-const fetchRemainingRelays = async (chainId: string, safeAddress: string): Promise<number | undefined> => {
+const fetchRemainingRelays = async (chainId: string, safeAddress: string) => {
```

Comment:
Nit: I would not remove the return type entirely.

```suggestion
const fetchRemainingRelays = async (chainId: string, safeAddress: string): Promise<number> => {
```